src/pages/projects.js

```diff
@@ -411,11 +412,12 @@ const ProjectDescriptions = () => (
         </div>
         <div className="project-description">
           <p>
-            Major:
+            Major:{" "}
             <a href="https://github.com/libbpf/libbpf">
               <b>libbpf</b>
-            </a>{" "}
-            Emerging:
+            </a>
+            <br>
+            Emerging:{" "}
```

Comment:
can we please also drop empty "Emergin: " label next to libbpf. I don't think we have any plans for other C/C++ libraries for now, it just looks weird.

Reply:
@anakryiko See PR #150

Reply:
But that is specific to XDP, right, so not about a library/loader dedicated around bpf syscall.. Maybe this is better represented in a different section somehow to avoid confusion.

Reply:
I was about to reply to that PR, but figured I might as well reply here since y'all are discussing it anyway: what's the best place to put libxdp, then? My reasoning for putting it next to libbpf is to make it obvious for users that it's an "XDP-specific layer on top of libbpf". Especially since we're now moving the AF_XDP functionality from libbpf to libxdp, I'm trying my best to make the transition as smooth as possible. Listing libxdp as "emerging" risks implying that we have demoted the support level of AF_XDP, for example. Adding a whole new "XDP Libraries" section is probably also a bit much as well, considering libxdp will be the only entry there?

Oh and TBH, even when counting the 20 or so contributors of the code we copied over from libbpf, I doubt we'll ever hit 50 contributors to libxdp unless we expand the scope significantly...

So, what to do? :)

Reply:
One thought could be to just add the paragraph of text describing libxdp at the end of the libbpf description, but don't list libxdp in the heading list of projects? I.e., merge only the second half of #150 ?